go/cli/check-tag/go.mod

```diff
@@ -0,0 +1,14 @@
+module github.com/foxglove/mcap/go/cli/check-tag
+
+go 1.18
+
+require github.com/foxglove/mcap/go/mcap v0.0.0-20221010221948-8b72814b2c50
```

Comment:
what's the meaning of this version specifier? will this have to be bumped when we make commits to the mcap library?